packages/valory/connections/abci/tests/test_abci.py

```diff
@@ -369,9 +402,23 @@ def setup(self) -> None:
             self.process_incoming_messages()
         )
 
+        # connection must be established,
+        # unlike TCP, only a single ECHO request is send with gRPC
+        time.sleep(5)
```

Comment:
Why 5?

Reply:
I'm not sure this is true (that we cannot fire up the image before starting the connecting), but I was informed by @0xArdi this was the case. Doesn't align with implementation in #1687 though where the autouse fixture fires before agents set and start up.